mmv1/templates/terraform/examples/cloudbuild_bitbucket_server_config_peered_network.tf.erb

```diff
@@ -0,0 +1,50 @@
+terraform {
+  required_providers {
+    google = {}
+  }
+}
+
+provider "google" {}
```

Comment:
any reason we need to set these explicitly?

Reply:
Nope. Good catch, removing them.